src/consensus.rs

```diff
@@ -399,7 +399,7 @@ impl<L, M> Consensus<L, M>
                              from,
                              leader_term);
                 self.transition_to_follower(leader_term, from, actions);
-                return self.append_entries_request(from, request, actions);
+                self.append_entries_request(from, request, actions);
```

Comment:
Yep.  I originally changed them to exclude the semicolon, but added it back in when I saw the method didn't have a return type.